src/Display/PointingModal/PointingModal.tsx

```diff
@@ -34,11 +34,15 @@ type PointingModal = React.FunctionComponent<Props> & {
 };
 
 PointingModal.Header = PointingModalHeader;
+PointingModal.Header.displayName = 'PointingModal.Header';
 PointingModal.Body = PointingModalBody;
+PointingModal.Body.displayName = 'PointingModal.Body';
 PointingModal.Item = PointingModalItem;
+PointingModal.Item.displayName = 'PointingModal.Item';
 PointingModal.Footer = PointingModalFooter;
+PointingModal.Footer.displayName = 'PointingModal.Footer';
```

Comment:
I think we also use `PointingModal.Footer` in the sample. Because I didn't know we have `Footer` until I saw this change. 😅

Reply:
Done